README.rst

```diff
@@ -1741,6 +1741,130 @@ If the request is made without passing the ``time`` argument, the server local t
 
 The ``time`` parameter was added to support `resilient collection and sending of data by the OpenWISP Monitoring Agent <https://github.com/openwisp/openwrt-openwisp-monitoring#collecting-vs-sending>`_.
 
+List wifi session
+#################
+
+.. code-block:: text
+   
+   GET /api/v1/monitoring/wifi-session/
+
+**Available filters**
+
+You can filter using ``device_id`` to get list of wifi sessions related to that particular device.
+
+.. code-block:: text
+   
+   GET /api/v1/monitoring/wifi-session/?device={device_id}
+
+You can filter using ``organization_id`` to get list of wifi sessions belongs to an organization.
+
+.. code-block:: text
+   
+   GET /api/v1/monitoring/wifi-session/?device__organization={organization_id}
```

Comment:
In OpenWISP, we use `organization_slug` for filtering. See https://github.com/openwisp/openwisp-controller#list-locations. It will be better to maintain this consistency, 

cc @nemesisdesign

Reply:
yes I think we should maintain consitency and add an equivalent filter